comixed-release/src/main/assembly/scripts/run.sh

```diff
@@ -20,14 +20,14 @@ realpath() {
   [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
 }
 
-ME=$(realpath $0)
-BINDIR=$(dirname ${ME})
-LIBDIR=$(realpath ${BINDIR}/../lib)
+ME=$(realpath "$0")
+BINDIR=$(dirname "${ME}")
+LIBDIR=$(realpath "${BINDIR}"/../lib)
 
 JAVA=$(which java)
 JAROPTIONS=''
 JVMOPTIONS=''
-COMIXED_JAR_FILE=${BINDIR}/comixed-app-*.jar
+COMIXED_JAR_FILE=$(echo "${BINDIR}"/comixed-app-*.jar)
```

Comment:
`BINDIR` needs to be enclosed in quotes to account for directory paths with spaces.  However, enclosing it in quotes means the `*` doesn't get expanded and becomes a literal.  Therefore I use the echo to escape `BINDIR`, then append the `*` for expansion.